docs/contributor-info/pull-requests.md

```
@@ -1,25 +1,26 @@
-* Ensure that docs are updated accordingly if a feature is added.
+- Provide changelog entry in the [unreleased section of the CHANGELOG.md](https://github.com/shakacode/react_on_rails/blob/master/CHANGELOG.md#unreleased).
+- Ensure CI passes and that you added a test that passes with the fix and fails without the fix.
+- Squash all commits down to one with a nice commit message _ONLY_ once final review is given. Make sure this single commit is rebased on top of master.
+- Please address all code review comments.
+- Ensure that docs are updated accordingly if a feature is added.
 
 ## Commit Messages
 
 From [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/)
 
 #### The seven rules of a great git commit message
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)